src/makeRobot.js

```diff
@@ -38,7 +38,84 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${
+        this.name}, chip version: ${
+        this.version}, wheels: ${
+        this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward() {
+      if (arguments[0] === undefined) {
+        arguments[0] = 1;
+      }
+
+      if (arguments[0] < 0) {
+        return this;
+      }
+
+      this.coords.y += arguments[0];
+
+      return this;
+    },
```

Comment:
```suggestion
    goForward(step = 1) {
      if (step > 0) {
        this.coords.y += step;
      }

      return this;
    },
```
You have only 1 number as a parameter for this function by the conditions.
It's enough to check, if it's not negative

Reply:
you right, good idea
thanks